From 039da673603e37abffb5e64f0c586ba848639d37 Mon Sep 17 00:00:00 2001
From: shiz01 <shiz01@tutanota.com>
Date: Sat, 18 Sep 2021 23:05:23 +0700
Subject: [PATCH] fix build - remove fetch content

Signed-off-by: shiz01 <shiz01@tutanota.com>
---
 CMakeLists.txt | 23 ++++++++++++-----------
 1 file changed, 12 insertions(+), 11 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index aa4cc31..279cb55 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,16 +5,17 @@ project(tl-function_ref VERSION 1.0.0 LANGUAGES CXX)
 option(FUNCTION_REF_ENABLE_TESTS "Enable tests." ON)
 
 include(FetchContent)
-FetchContent_Declare(
-  tl_cmake
-  GIT_REPOSITORY https://github.com/TartanLlama/tl-cmake.git
-)
-FetchContent_GetProperties(tl_cmake)
-if(NOT tl_cmake_POPULATED)
-  FetchContent_Populate(tl_cmake)
-  set(CMAKE_MODULE_PATH ${tl_cmake_SOURCE_DIR} ${CMAKE_MODULE_PATH})
-endif()
-include(add-tl)
+#FetchContent_Declare(
+#  tl_cmake
+#  GIT_REPOSITORY https://github.com/TartanLlama/tl-cmake.git
+#)
+#FetchContent_GetProperties(tl_cmake)
+#if(NOT tl_cmake_POPULATED)
+#  FetchContent_Populate(tl_cmake)
+#  set(CMAKE_MODULE_PATH ${tl_cmake_SOURCE_DIR} ${CMAKE_MODULE_PATH})
+#endif()
+
+include(add-tl.cmake)
 
 tl_add_library(function-ref SOURCES 
                include/tl/function_ref.hpp)
@@ -37,4 +38,4 @@ if(FUNCTION_REF_ENABLE_TESTS)
   target_link_libraries(tests Catch function-ref)
 
   set_property(TARGET tests PROPERTY CXX_STANDARD 14)
-endif()
\ No newline at end of file
+endif()
-- 
2.32.0

